### PR TITLE
TFA-61 improve HTML and css

### PR DESCRIPTION
**Pull request by lestairon**
## Issue Link
https://edgaro.atlassian.net/browse/TFA-61
https://edgaro.atlassian.net/browse/TFA-57
https://edgaro.atlassian.net/browse/TFA-58

## Checklist:

- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed locally.

### CODE REVIEW

File: src/components/Board.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React, { useEffect } from "react";
 import Card from "./Card";
 import Counter from "./Counter";
+import Timer from "./Timer";
 import Menu from "./Menu";
 import PropTypes from "prop-types";
 import { useSelector, useDispatch } from "react-redux";
-import styled from "styled-components";
+import { BoardDiv, CardContainerDiv } from "./components";
```
Comment (EdgarOrtegaRamirez) [inline src/components/Board.js]: why do you add the suffix `Div` to these components with?

File: src/components/Counter.js [hunk 2]
```diff
@@ -1,11 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import styled from "styled-components";
-
-const CounterH3 = styled.h3`
-  width: 2vw;
-  float: left;
-`;
+import { CounterH3 } from "./components";
```
Comment (EdgarOrtegaRamirez) [inline src/components/Counter.js]: Why did you add `H3`? Why do you need to specify that?

File: src/components/ScoreBoard.js [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
-`;
-
 const ScoreBoard = () => {
   const [score, setScore] = useState([]);
 
   useEffect(() => {
-    const data = firestoreDB.collection("scores");
-    data.onSnapshot(QuerySnapshot => {
+    const data = firestoreDB.collection("scores").limit(3);
+    const unsub = data.onSnapshot(QuerySnapshot => {
```
Comment (EdgarOrtegaRamirez) [inline src/components/ScoreBoard.js]: `unsub`? please use a name more appropriate

File: src/components/ScoreBoard.js [hunk 4]
```diff
@@ -1,18 +1,13 @@
...
-  top: 0;
-`;
-
 const ScoreBoard = () => {
   const [score, setScore] = useState([]);
 
   useEffect(() => {
-    const data = firestoreDB.collection("scores");
-    data.onSnapshot(QuerySnapshot => {
+    const data = firestoreDB.collection("scores").limit(3);
```
Comment (EdgarOrtegaRamirez) [inline src/components/ScoreBoard.js]: `data` is too generic, use a better name

File: src/components/__tests__/integration/board.test.js [hunk 5]
```diff
@@ -29,46 +28,51 @@ const component = (
...
+    const cards = container.firstChild.firstChild.nextSibling;
+    fireEvent.click(cards.firstChild);
+    fireEvent.click(cards.lastChild);
+    fireEvent.click(cards.firstChild.nextSibling);
+    expect(cards.firstChild.nextSibling.innerHTML).toBe("");
   });
 
   it("shows the card for a short span of time if incorrect", async () => {
     jest.useFakeTimers();
     const { container } = component(<Board numberOfCards={2} order={true} />);
```
Comment (EdgarOrtegaRamirez) [inline src/components/__tests__/integration/board.test.js]: i know this change is not related to this PR, but I still dont understand what `order={true}` does..., could you please explain?

**Review by EdgarOrtegaRamirez — CHANGES_REQUESTED** [review]

File: src/components/__tests__/integration/board.test.js (hunk 5 above)
Comment (lestairon) [reply src/components/__tests__/integration/board.test.js]: This prop is there for the sake of testing.
When is set to true it doesn't shuffle the cards. They appear in `order`

Comment (EdgarOrtegaRamirez) [reply src/components/__tests__/integration/board.test.js]: that logic does not make sense to me. when `order=true` there is no order? is that correct?

Comment (lestairon) [reply src/components/__tests__/integration/board.test.js]: No, when `order=true` it does not suffle the cards. They appear in order

Comment (lestairon) [reply src/components/__tests__/integration/board.test.js]: Else, the cards are shuffled randomly

Comment (EdgarOrtegaRamirez) [reply src/components/__tests__/integration/board.test.js]: still does not make sense, shouldn't it be `shuffle=true|false`?
